cloudlink-function-mapper-desktop/src/main/java/com/gluonhq/functionmapper/views/MainController.java

```diff
@@ -110,9 +110,13 @@ private void search(String tag) {
         reset();
         var searchStackOverflow = getApp().service().searchStackOverflow(tag);
         searchStackOverflow.setOnSucceeded(e -> {
-            listView.setItems(FXCollections.observableArrayList(searchStackOverflow.get().getItems()));
-            if (top.getChildren().size() == 3) {
-                top.getChildren().remove(2);
+            var stackResponse = searchStackOverflow.get();
```

Comment:
This breaks Java 9, do we really need var?

Reply:
This particular sample is for Java 11+.